pkg/upgrade/upgrades/v25_1_add_jobs_tables.go

```diff
@@ -182,9 +212,6 @@ func backfillJobsTablesAndColumns(
 		}
 
 		jobsBackfilled += done
```

Comment:
perhaps we could logEvery here, so have some notion of progress made at least in the logs. I suppose we could also arm tse with your todo/done query.

Reply:
yeah, I think the todo query is fine. Or we'll add progress updates in a followup if we find this being needed often.

Reply:
big +1 on adding incremental logging for this job.